client/src/components/Dashboard/DashboardForecast/DashboardForecast.utils.ts

```diff
@@ -0,0 +1,93 @@
+import { Dispatch } from 'redux';
+
+// Store
+import { RootState } from 'redux/store';
+import {
+  getLocationHourlyWeather,
+  getLocationThreeHourlyWeather,
+  getLocationDailyWeather,
+  getLocationHourlyAirQuality,
+  getLocationDailyAirQuality
+} from 'redux/actionCreators/location';
+
+export enum WeatherType {
+  hourly = 'hourly',
+  threeHourly = 'threeHourly',
+  daily = 'daily'
+}
+
+export enum AirQualityType {
+  hourly = 'hourly',
+  daily = 'daily'
+}
+
+export type ForecastType = WeatherType | AirQualityType;
+
+export const selectLocationWeather = (
+  { location: { weather } }: RootState,
+  selectedForecastType: WeatherType | AirQualityType
```

Comment:
please replace `WeatherType | AirQualityType` in this file too

Reply:
Sorry, I forgot